services/shhext/history.go

```diff
@@ -353,8 +353,8 @@ func (filter BloomFilterOption) ToMessagesRequestPayload() ([]byte, error) {
 		Bloom: filter.Filter,
 		// Client must tell the MailServer if it supports batch responses.
 		// This can be removed in the future.
-		Batch: true,
-		Limit: 10000,
+		//Batch: true,
```

Comment:
why is this commented out and limit is changed?

Reply:
you are right, thanks. i was experimenting with it and thought that i discarded changes